### PR TITLE
Math Magicians: setup

### PR DESCRIPTION
**Pull request by Nuga0909**
In this pull request, I:
- Set up a React app.
- Use React documentation.

### CODE REVIEW

**Review by DammyShittu — COMMENTED** [review]
Hi @Nuga0909,

While you made a great effort in this project, I cannot proceed to review your code.

## Invalid Code Review Request

### Kindly customize your readme documentation.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: README.md [hunk 1]
```diff
@@ -1,70 +1,120 @@
...
+
+## Acknowledgments
+
+- Hat tip to Microverse for giving me this project to work on
+- Hat tip to the person his/her code was used
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./MIT.md) licensed.
+
```
Comment (Olamarx) [inline README.md:119]: - [x] I would encourage you to fix this MIT link since it is on it you get permission to use this repository.

**Review by Olamarx — CHANGES_REQUESTED** [review]
Hi @Nuga0909, 

Good job so far, indeed you have spent time on this project!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
- Linters passed ✔️
- Readme file professional ✔️
- CRA usage ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (Nuga0909) [reply README.md:119]: fixed

**Comment by Nuga0909** [comment]
I merged the pull request by mistake, as i thought the last reviewer had approved..

He only requested optional changes.

**Review by Gambit142 — COMMENTED** [review]
Hi @Nuga0909 👋,  

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

- You have submitted a merged pull request. You were not meant to merge this pull request. Kindly create a new pull request and provide the link to @Olamarx review in it. 👍 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Pull request merged**